app.js

```diff
@@ -0,0 +1,22 @@
+const port = 5000;
```

Comment:
You aren't making use of this `port` anywhere else in your code. You can utilize it on the `app.listen()` at the bottom of this file